vulnfeeds/cmd/alpine/main.go

```diff
@@ -41,6 +41,7 @@ func main() {
 	outputBucketName := flag.String("output_bucket", outputBucketDefault, "The GCS bucket to write to.")
 	numWorkers := flag.Int("num_workers", 64, "Number of workers to process records")
 	uploadToGCS := flag.Bool("uploadToGCS", false, "If true, do not write to GCS bucket and instead write to local disk.")
+	syncDeletions := flag.Bool("sync_deletions", true, "If false, do not delete files in bucket that are not local")
```

Comment:
Let's keep this consistently false, and tweak it in either the script or the kubernetes file.